app/services/api/v1x0/catalog/add_to_order.rb

```diff
@@ -13,6 +13,13 @@ def process
           @order_item = order.order_items.create!(order_item_params.merge!(:service_plan_ref => service_plan_ref))
           @order_item.update_message("info", "Order item tracking ID (x-rh-insights-request-id): #{@order_item.insights_request_id}")
 
+          # In the earlier design tracking (insights-request-id) is only available in order item since an order can have multiple
```

Comment:
@bzwei The request_id is generated by 3scale when a UI submits an Order. So it encompasses all things under the order. An order can have multiple order items but they would all have the same request_id. So even if we added some extra order_items they should all have the same request_id. When a user looks at the Progress Message in the UI we need to see the request_id for debugging purpose. Currently in order to get to the request_id you have to  go to the Provision tab.

Reply:
To create an order, add order_items to an order, and start the order involves a few REST calls. So multiple request_ids are created by 3scale during the course. We have picked the request_id when the product order_item is created/added to the order as the tracking_id for the order and all order_items under the umbrella.